macos/Sources/Features/QuickTerminal/QuickTerminalController.swift

```diff
@@ -306,6 +306,9 @@ class QuickTerminalController: BaseTerminalController {
 
     private func syncAppearance(_ config: Ghostty.Config) {
         guard let window else { return }
+        
+        // Update the quick terminal size right away
+        config.quickTerminalSize.apply(window, config.quickTerminalPosition)
```

Comment:
[Suggestion] Update the window size here because the call is delayed until the quick terminal becomes visible, which is too late. Not ideal, since this function is called repeatedly until it can be consumed. How about synchronizing the appearance when `toggle` is called?

Reply:
addressed in #4055